src/main/scala/it/pagopa/interop/partyprocess/service/impl/UserRegistryManagementServiceImpl.scala

```diff
@@ -21,7 +21,7 @@ final case class UserRegistryManagementServiceImpl(invoker: UserRegistryManageme
 ) extends UserRegistryManagementService {
   implicit val logger: Logger = LoggerFactory.getLogger(this.getClass)
 
-  private val userFields2fetch: Seq[String] = Seq("name", "familyName", "fiscalCode")
+  private val userFields2fetch: Seq[String] = Seq("name,familyName,fiscalCode")
```

Comment:
For some reason of this list the generated code is sending just the "fiscalCode" value

Reply:
[Here](https://github.com/pagopa/interop-selfcare-proxy-clients/pull/4) you can find a solution, that must be discussed yet.

Reply:
thanks @beetlecrunch , I'll watch the result of that review and replicate it here!